releasenotes/notes/28915.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: release-notes/v2
+kind: feature
+area: istioctl
+releaseNotes:
```

Comment:
If there is no issue created for this PR, make sure to add the PR number here so it won't miss in release note generation.
```
issue:
   - 31795
```